modules/src/ics04_channel/packet.rs

```diff
@@ -112,6 +113,14 @@ pub struct Packet {
     pub timeout_timestamp: Timestamp,
 }
 
+impl Packet {
+    pub fn timed_out(&self, dst_chain_height: Height) -> bool {
+        self.timeout_height != Height::zero() && self.timeout_height < dst_chain_height
+            || self.timeout_timestamp != Timestamp::none()
+                && Timestamp::now().check_expiry(&self.timeout_timestamp) == Expired
+    }
```

Comment:
The order of precedence here is not obvious. Could we improve by making precedence explicit? I am guessing it should be like this:

```suggestion
    pub fn timed_out(&self, dst_chain_height: Height) -> bool {
        (self.timeout_height != Height::zero() && self.timeout_height < dst_chain_height)
            || (self.timeout_timestamp != Timestamp::none()
                && Timestamp::now().check_expiry(&self.timeout_timestamp)) == Expired
    }
```

Reply:
b08c348b